tests/qos/test_buffer.py

```diff
@@ -980,7 +1039,10 @@ def test_change_speed_cable(duthosts, rand_one_dut_hostname, conn_graph_facts, p
                             new_size = pg_size,
                             new_pg_number = 1)
 
-            check_pfc_enable(duthost, port_to_test, '6')
+            if duthost.facts['asic_type'] == 'barefoot':
```

Comment:
let's define a variable like `additional_pg_to_test` according to vendor and reference it in this function. by doing so we do not need to check vendor every time.

Reply:
OK, it makes sense. Thank you very much!